.github/workflows/docs.yml

```diff
@@ -21,11 +21,21 @@ on:
 jobs:
   sphinx:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, "3.10"]
+
     steps:
     - uses: actions/checkout@v1
-    - uses: josh146/sphinx-action@master
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
       with:
-        docs-folder: "doc/"
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        pre-build-command: "apt install -y graphviz && pip3 install -r doc/requirements.txt && pip3 install . pip3 install openfermionpyscf"
-        build-command: "sphinx-build -b html . _build -W --keep-going"
```

Comment:
How come Sphinx action is being deleted? This will cause annotation of the documentation to stop happening 🙂

Reply:
As mentioned in comment https://github.com/PennyLaneAI/pennylane/pull/2824#discussion_r923892875, these changes are just to test the doc/requirements.txt using different python versions on Linux. They will be removed before merging.

Reply:
Thanks for clarifying @AlbertMitjans :)